charts/steadybit-extension-kubernetes/templates/deployment.yaml

```diff
@@ -143,6 +143,10 @@ spec:
             {{- end }}
             - name: STEADYBIT_EXTENSION_DISCOVERY_MAX_POD_COUNT
               value: "{{ .Values.discovery.maxPodCount }}"
+            {{- if .Values.discovery.disableArgoRollouts }}
```

Comment:
Could you add a test to `charts/steadybit-extension-kubernetes/tests/deployment_test.yaml` -> `manifest should match snapshot with disabled discoveries`